svelte-motoko-starter/README.md

```diff
@@ -0,0 +1,190 @@
+<p align="left" >
+  <img width="240"  src="./assets/logo.png">
+</p>
+
+# Svelte Dapp with Motoko & Internet Identity
+
+This repository is meant to give Svelte developers an easy on-ramp to get started with developing decentralized applications (Dapps in short) for the Internet Computer blockchain. Dapps, also known as smart contracts are specialized software that run on a blockchain.
```

Comment:
Drive-by comment: Perhaps provide a link for Svelte.

Reply:
Thanks, added a link